package.json

```diff
@@ -4,11 +4,26 @@
   "description": "",
   "main": "cypress.config.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "lint": "eslint .",
```

Comment:
It means pre-commit will format files using prettier but only check them using eslint?

Reply:
Nevermind, I saw comment below